src/main/java/frc/robot/RobotContainer.java

```diff
@@ -21,10 +24,31 @@
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
 
   private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
+  LogitechGamingPad drivePad;
+
+  DriveTrain driveTrain;
+  Shooter shooter;
+  Intake intake;
```

Comment:
```suggestion
  Intake intake;
```
Since this branch deals solely w/ the Intake subsystem, it is okay to remove the other 2 subsystems. The branches that focus on the drivetrain and shooter will declare and initialize their respective subsystems.